README.md

```diff
@@ -180,6 +180,10 @@ These settings are specific to VSCodeVim.
     }
 ```
 
+#### `"vim.visualstar"`
+* When pressing `*` or `#` in visual mode treat it as a new search
```

Comment:
Could we rephrase this slightly:

"In visual mode, start a search with * or # using the current selection."